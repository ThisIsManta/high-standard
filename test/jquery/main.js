import x from 'jquery'

void window.jquery
