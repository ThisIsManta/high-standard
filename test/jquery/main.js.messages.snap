error 1:8 [levitate/import-convention] Expected the default import to be "$".
error 1:8 [no-unused-vars] 'x' is defined but never used.